src/util.h

```diff
@@ -497,6 +497,14 @@ void UnsetArg(const std::string &strArg);
  */
 void SetBoolArg(const std::string &strArg, bool fValue);
 
+/*
+ * Convert string into true/false
+ *
+ * @param strValue String to parse as a boolean
+ * @return true or false
+ */
+bool InterpretBool(const std::string &strValue);
```

Comment:
I'm cool to expose this to the rest of the code base.

Do you have some use case in mind already?